.gitignore

```diff
@@ -3,3 +3,4 @@ polkadot-deployer.benchmark*.json
 polkadot-deployer.benchmark*.gnuplot
 *.png
 terraform/**/.terraform
+.DS_Store
```

Comment:
I think it should be better to keep in the .gitignore file entries that will be common to all the people using the repo, we can have our customized ignore at the home dir level, for instance, wdyt?
```suggestion
```